include/libsxg/internal/sxg_codec.h

```diff
@@ -21,35 +21,45 @@
 #include <openssl/sha.h>
 #include <stdbool.h>
 
-#include "libsxg/sxg_buffer.h"
-
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-// Replaces contents of `dst` with SHA-hash of `src`. Returns true on success.
-bool sxg_calc_sha256(const sxg_buffer_t* src, sxg_buffer_t* dst);
-bool sxg_calc_sha384(const sxg_buffer_t* src, sxg_buffer_t* dst);
+// Returns size of expected digest length (fixed).
+inline size_t sxg_sha256_size() { return SHA256_DIGEST_LENGTH; }
```

Comment:
Are these functions necessary anyway? You don't use them except for tests.

Reply:
Removed.